beancount_web/util/dateparser.py

```diff
@@ -40,8 +40,9 @@ def daterange(year=None, month=None, day=None, timedelta=datetime.timedelta()):
     """
     year, month, day = map(lambda x: int(x) if x else None, (year, month, day))
     if not (year or month or day):
-        date = datetime.date.today() + timedelta
-        return date, date
+        start = datetime.date.today() + timedelta
+        end = start + datetime.timedelta(days = 1)
```

Comment:
Adding 23:59:59 would have been more correct, but apparently only `date` is considered, `time` is ignored.